build.sbt

```diff
@@ -276,7 +278,8 @@ lazy val observe_server     = project
         TestLibs.value,
         PPrint.value
       ) ++ MUnit.value ++ Http4s ++ Http4sClient ++ PureConfig ++ SeqexecOdb ++ Monocle.value ++ WDBAClient ++
-        Circe.value
+        Circe.value,
+    headerSources / excludeFilter := HiddenFileFilter || (file("modules/server") / "src/main/scala/pureconfig/module/http4s/package.scala").getName
```

Comment:
This is because we are using a copy of the pureconfig-http4s bridge?

Reply:
Yes, and I did not want to stamp the AURA license on "borrowed" code.